inlong-dataproxy/dataproxy-docker/Dockerfile

```diff
@@ -29,6 +29,9 @@ ENV MANAGER_OPENAPI_PORT=8083
 ENV AUDIT_PROXY_URL=127.0.0.1:10081
 # pulsar or tubemq
 ENV MQ_TYPE=pulsar
+ENV CLUSTER_TAG=''
+ENV CLUSTER_NAME=default
+ENV CLUSTER_IN_CHARGES=admin
```

Comment:
keep the default same with common.properties
```
proxy.cluster.name=default_dataproxy
proxy.cluster.tag=default_cluster
proxy.cluster.inCharges=admin
```

Reply:
done